README.md

```diff
@@ -83,6 +77,14 @@ with datadog_api_client.v1.ApiClient(configuration) as api_client:
         print("Exception when calling AWSIntegrationApi->create_aws_account: %s\n" % e)
 ```
 
+## Getting Started
+
+<!--- we need to add a working getting started example here, but which one? Do we know what the most used endpoint is? In datadogpy, we do sending an event, which would work too. Or we can add more than one. --->
```

Comment:
We can't send event because we don't have the endpoint :). The most used API is probably `GetMonitor`? That would be much better than the AWS integration for sure.